guides/micronaut-data-jdbc-repository/kotlin/src/main/kotlin/example/micronaut/domain/Genre.kt

```diff
@@ -0,0 +1,13 @@
+package example.micronaut.domain
+
+import io.micronaut.data.annotation.GeneratedValue
+import io.micronaut.data.annotation.Id
+import io.micronaut.data.annotation.MappedEntity
+
+@MappedEntity
+data class Genre(
+    @field:Id
+    @field:GeneratedValue(GeneratedValue.Type.AUTO)
```

Comment:
Mironaut data docs for Kotlin does not use `@field:Id`but just `@GeneratedValue`